scalardb-kotlin-sample/build.gradle.kts

```diff
@@ -21,13 +21,8 @@ tasks.test {
     useJUnitPlatform()
 }
 
-tasks.withType<KotlinCompile> {
-    kotlinOptions.jvmTarget = "1.8"
-}
-
-java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Because there is `kotlin { jvmToolchain(8) }`, this setting seems to be unnecessary.